documentation/docs/20-core-concepts/50-state-management.md

```diff
@@ -140,7 +140,9 @@ When you navigate around your application, SvelteKit reuses existing layout and
 <div>{@html data.content}</div>
 ```
 
-...then navigating from `/blog/my-short-post` to `/blog/my-long-post` won't cause the component to be destroyed and recreated. The `data` prop (and by extension `data.title` and `data.content`) will change, but because the code isn't rerunning, `estimatedReadingTime` won't be recalculated.
+...then navigating from `/blog/my-short-post` to `/blog/my-long-post` won't cause the components (layout and page) to be destroyed and recreated. Instead the `data` prop (and by extension `data.title` and `data.content`) will reactively change (as it would with a regular Svelte component) and, because the code isn't rerunning, `estimatedReadingTime` won't be recalculated and lifecycle methods like `onMount` and `onDestroy` won't rerun.
+
+> If your code in `onMount` and `onDestroy` has to run again after navigation you can use [aferNavigate](modules#$app-navigation-afternavigate) and [beforeNaviate](modules#$app-navigation-beforenavigate) respectively.
 
 Instead, we need to make the value [_reactive_](https://learn.svelte.dev/tutorial/reactive-assignments):
```

Comment:
This sentence is referring to the `estimatedReadingTime` variable, the flow is broken by the change.